test/integration-server/acrolinx-endpoint.test.ts

```diff
@@ -482,6 +482,28 @@ describe('e2e - AcrolinxEndpoint', () => {
       });
     });
 
+    //Acrolinx Live target needs to be assigned for the following test case 
+    describe('liveSearch', () => {
+      it.skip('should get suggestions for a live search', async () => {
+        const liveRequest = {
+          'request-id': 'abc1-Request',
+          phrase: 'phrase from the document',
+          target: '79ec4ac3-a9e3-4532-9d01-ae6141aefea2'
```

Comment:
hardcoded target id? Can checking capabilities help us identify the AcrolinxLive capable target?

Reply:
This is a good idea! I can update this.